cypress/e2e/about.cy.ts

```diff
@@ -45,7 +45,7 @@ describe('About page', () => {
         })
 
         it('shows member details', () => {
-            cy.get('[data-testid="member-avatar"]').should('exist')
+            cy.get('[data-testid="member-avatar-0"]').should('exist')
```

Comment:
This tests each avatar of the members their own  unique id and can also be tested in other files when images are used in the future using the prop.